### PR TITLE
Added projects to dataset lists

### PR DESCRIPTION
**Pull request by Samad198**
# Changes
- Added projects to dataset lists

# Migrations

# Screenshots

image

image

# Checks

**Important:** please complete these **before** merging.
- [ ] Run migrations, if any.
- [ ] Update `changelog.md`, including migration instructions if any.
- [ ] Run unit tests.

### CODE REVIEW

File: api/mapping/views.py [hunk 1]
```diff
@@ -215,11 +216,15 @@ class ProjectListView(ListAPIView):
     filterset_fields = {"name": ["in", "exact"]}
 
     def get_serializer_class(self):
+        print(self.request.GET)
```
Comment (dcl10) [inline api/mapping/views.py:219]: remove

File: api/mapping/views.py [hunk 2]
```diff
@@ -215,11 +216,15 @@ class ProjectListView(ListAPIView):
...
         if (
             self.request.GET.get("name") != None
             or self.request.GET.get("name__in") != None
         ):
             return ProjectSerializer
+        if self.request.GET.get("datasets") != None:
```
Comment (dcl10) [inline api/mapping/views.py:225]: same for the above lines too, I think PEP8 prefers you to test if something is `None` by doing `my_variable is not None` as opposed to `!= None`

File: react-client-app/src/components/UploadScanReport.jsx [hunk 3]
```diff
@@ -41,7 +41,7 @@ const UploadScanReport = ({ setTitle }) => {
         const dataPartnerQuery = await useGet("/datapartners/")
         setDataPartners([{ name: "------" }, ...dataPartnerQuery.sort((a, b) => a.name.localeCompare(b.name))])
         setLoadingMessage(null)
-        const projectsQuery = await useGet("/projects/")
+        const projectsQuery = await useGet("/projects/?datasets=true")
```
Comment (dcl10) [inline react-client-app/src/components/UploadScanReport.jsx:44]: can you make a note to do this in the changelog please?

**Review by dcl10 — COMMENTED** [review]
LGTM - don't forget the changelog!

**Review by dcl10 — APPROVED** [review]

**Review by spco — APPROVED** [review]

**Pull request merged**